web/.eslintrc.json

```diff
@@ -17,6 +17,8 @@
         }
       }
     ],
+    "jsx-a11y/label-has-associated-control": ["error", { "assert": "either" }],
```

Comment:
This `label-has-associated-control` rule is new, and it replaces an older deprecated rule called `jsx-a11y/label-has-for`.  The default behavior is to require `<label>` elements to both wrap the inputs they apply to ***AND*** have a `for` attribute.  We felt like that was overkill (and also difficult), so we modified the older rule.  This is just updating our config to similarly modify the new rule.  The old rule is deprecated but still in force, so I couldn't remove our modification for it yet.

Reply:
Thanks for the clarification!